pkgs/os-specific/darwin/apple-source-releases/xnu/default.nix

```diff
@@ -73,7 +84,7 @@ appleDerivation' (if headersOnly then stdenvNoCC else stdenv) ({
     cat > sdk/usr/local/libexec/availability.pl <<EOF
       #!$SHELL
       if [ "\$1" == "--macosx" ]; then
-        echo 10.0 10.1 10.2 10.3 10.4 10.5 10.6 10.7 10.8 10.9 10.10 10.11
+        echo 10.0 10.1 10.2 10.3 10.4 10.5 10.6 10.7 10.8 10.9 10.10 10.11 10.12 10.13 10.14 10.15 11.0
```

Comment:
Are we sure this causes no problems for x86_64-darwin, since that doesn't support any versions past 10.12?

Reply:
I am unsure about this, so I retained the old versions for x86_64-darwin in the new commit.